transmit-lib/src/cmr/transmit/access_control.clj

```diff
@@ -122,6 +126,18 @@
                                        :accept :json}
                                       http-options)}))))
 
+(defn get-current-sids
```

Comment:
We need to performance test this before we go to operations with this change. If it is more than 50 ms in SIT or UAT we should investigate.